README.md

```diff
@@ -80,7 +80,7 @@ Next, change the following specifications in `docker-compose.yaml`:
     nginx:
         image: jonasal/nginx-certbot:dev
         volumes:
-            - ./nginx/localhost_conf.d:/etc/nginx/user_conf.d
```

Comment:
revert

Reply:
done, will need to relook into it in another branch